packages/test-support/package.json

```diff
@@ -41,6 +41,9 @@
     "tmp": "^0.2.1"
   },
   "devDependencies": {
+    "ember-cli-4.4": "npm:ember-cli@~4.4.0",
```

Comment:
why do we need to specify npm here?
eg. 
```
"ember-cli": "^4.4.0",
"ember-data": "^4.4.0",
"ember-source": "^4.4.0"
```

Reply:
Changed